src/DSharp.Compiler/Utility.cs

```diff
@@ -32,7 +32,7 @@ internal static class Utility
             "enum", "implements", "package", "super", "byte", "export",
             "import", "private", "synchronized", "char", "extends", "int",
             "protected", "throws", "class", "sealed", "interface", "public",
-            "const", "float", "long", "short",
+            "const", "float", "long", "short", "arguments",
```

Comment:
missing `instanceof` in the list

Reply:
It is actually on the list 😄 the diff doesn't show the whole file